csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -5572,6 +5574,36 @@ A distributing party satisfies the "CSAF aggregator" role if the party:
 > CSAF aggregators can be provided for free or as a paid service.
 > To aid in automation, CSAF aggregators MAY mirror CSAF documents from CSAF publishers. Regarding the terms of use they SHOULD consult with the issuing party.
 
+## 7.3 Retrieving rules
+
+The retrieving process executes in two phases: Finding the `provider-metadata.json` (requirement 7 in section 7.1) and retrieving CSAF documents.
+
+> A retrieving party MAY choose to do the first phase only when adding new or updating distributing parties. However, it is recommended to check regulary whether new information is available.
```

Comment:
Again what about _SHOULD look for and download the `provider_metadata.json` earch time._?

Reply:
The question is: Is that really necessary? In most cases not. Therefore, I'm a little bit reluctant to make such hard statement...

Reply:
See [f2e97a8](https://github.com/oasis-tcs/csaf/pull/430/commits/f2e97a86f6881e08915b689922fa9548640dfd1b) for my suggestion.